app/views/posts/show.html.erb

```diff
@@ -1,18 +1,27 @@
-
 <!DOCTYPE html>
 <html>
 <head>
-  <title>My Blog Post</title>
+  <title><%= @post.title %></title>
 </head>
 <body>
-  <h1>Welcome to My Blog Post</h1>
+  <h1><%= @post.title %></h1>
+
+  <p><%= @post.text %></p>
+
+  <p><strong>Author:</strong> <%= @post.author.name %></p>
+
+  <p><strong>Likes:</strong> <%= @post.likes_count %></p>
   
-  <div>
-    <!-- Your blog post content goes here -->
-  </div>
+  <p><strong>Comments:</strong> <%= @post.comments_count %></p>
+
+  <h2>Comments:</h2>
+  <% @post.ordered_comments.each do |comment| %>
+    <div>
+      <p><strong><%= comment.user.name %>:</strong> <%= comment.text %></p>
+    </div>
+  <% end %>
+
+  <p><%= link_to "Back to All Posts", user_posts_path(@post.author_id) %></p>
```

Comment:
- [ ] It is nice to keep to design guidelines, kindly match your UI to the given wireframe
in your design :point_down: 
![Screenshot from 2023-11-25 11-51-58](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/5c583855-c726-468b-a4d8-0227e7e87d2e)
given wireframe :point_down: 
![Screenshot from 2023-09-01 21-07-50](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/408610f0-be6f-4456-93d1-d09959e8f65b)